calm_adapter/calm_adapter/src/test/scala/uk/ac/wellcome/calm_adapter/CalmStoreTest.scala

```diff
@@ -10,39 +11,44 @@ import uk.ac.wellcome.storage.{StoreReadError, Version}
 class CalmStoreTest extends FunSpec with Matchers {
 
   type Key = Version[String, Int]
-  type Data = Map[String, String]
+
+  val retrievedAt = Instant.ofEpochSecond(123456)
 
   it("stores new CALM records") {
     val data = dataStore()
-    val record = CalmRecord("A", Map("key" -> "value"))
+    val record = CalmRecord("A", Map("key" -> "value"), retrievedAt)
     calmStore(data).putRecord(record) shouldBe Right(Some(Version("A", 0)))
-    data.entries shouldBe Map(
-      Version("A", 0) -> Map("key" -> "value")
-    )
+    data.entries shouldBe Map(Version("A", 0) -> record)
   }
 
-  it("stores already seen CALM records when the data has changed") {
-    val data = dataStore(Map(Version("A", 1) -> Map("key" -> "old")))
-    val record = CalmRecord("A", Map("key" -> "new"))
-    calmStore(data).putRecord(record) shouldBe Right(Some(Version("A", 2)))
+  it("stores already seen CALM records when newer data") {
+    val oldRecord = CalmRecord("A", Map("key" -> "old"), retrievedAt)
+    val newRecord = CalmRecord(
+      "A",
+      Map("key" -> "new"),
+      Instant.ofEpochSecond(retrievedAt.getEpochSecond + 2))
```

Comment:
minor: just to make these two line up neatly, maybe pull the times into variables?

```scala
val oldTime = retrievedAt
val newTime = Instant.ofEpochSecond(retrievedAt.getEpochSecond + 2)

val oldRecord = CalmRecord("A", Map("key" -> "old"), oldTime)
val newRecord = CalmRecord("A", Map("key" -> "old"), newTime)
```

Same meaning, but the similarity between oldRecord/newRecord is more apparent.